### DISCUSSION TITLE
The system cannot find the path specified

### DISCUSSION THREAD

**Discussion by sambitNOOB** [Difegue/LANraragi#924] [Q&A]
**LRR Version and OS**
Please tell which version you're running
idk, must be the latest as I downloaded it from Microsoft store 
as well as your OS and installation method. (Windows, Docker, Native)
OS BUILD :19045.3693

**Bug Details**
downloaded it, had wsl1 installed first and then when I added the folder in E:/ksk archive, it doesnt seems to trace the path, idk what to do, I dont know how to code and stuff 
**Matching Logs**
```
Executing the following command on WSL: export LRR_FORCE_DEBUG=1 && export LRR_NETWORK=http://*:3000 && export LRR_DATA_DIRECTORY='/mnt/e/ksk archive' && export LRR_THUMB_DIRECTORY='/mnt/e/ksk archive' && cd /home/koyomi/lanraragi && rm -f public/temp/server.pid && mkdir -p log && mkdir -p content && mkdir -p database && sysctl vm.overcommit_memory=1 && redis-server /home/koyomi/lanraragi/tools/build/docker/redis.conf --dir '/mnt/e/ksk archive/' --daemonize yes && perl ./script/launcher.pl -f ./script/lanraragi
The system cannot find the path specified.
```

**Screenshots**
please help brother, I dont know to code, wsl and stuff

![Screenshot 2023-11-19 173134](https://github.com/Difegue/LANraragi/assets/84120515/c56b7e30-9358-4be1-b9ed-0a88e7a8edfa)
![Screenshot 2023-11-19 173220](https://github.com/Difegue/LANraragi/assets/84120515/6421f7af-61da-41b2-868f-7b20837778a0)
![Screenshot 2023-11-19 173251](https://github.com/Difegue/LANraragi/assets/84120515/607f549b-718f-4997-a82b-fdf62a326241)
![Screenshot 2023-11-19 173344](https://github.com/Difegue/LANraragi/assets/84120515/1afcdec7-d657-4e36-92dd-55ed266a5d0b)

**Comment by Difegue** [comment]
Hm this is a bit old, sorry for the delay -- The MS Store version is a bit outdated these days since Microsoft won't let me push an update to it without paying for codesigning..  

I'd recommend trying a recent windows nightly. (Windows 0.9.0 has some issues of its own)
